docs/relying-party-integration.md

```
@@ -0,0 +1,1011 @@
+# Development and Integration with eSignet
+
+This guide provides a step-by-step approach for developers who want to integrate their application as a **Relying Party (RP)** with **eSignet** (Go backend, v2.0.0+).
+
+> **Migrating from the Java-based eSignet?** See [Changes from the Java Implementation](#changes-from-the-java-implementation) at the bottom — several endpoint paths and behaviours have changed.
+
+### Prerequisites
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)